tests/test_autocompleter.py

```diff
@@ -50,4 +53,28 @@ def test_answer(term_index: TermIndex, string: str, suggestions: list[list[str]]
 
     response = autocompleter.suggestions(string)
 
-    assert { tuple(x) for x in response } == { tuple([Term(value=s) for s in xs]) for xs in suggestions}
\ No newline at end of file
+    assert { tuple(x) for x in response } == { tuple([Term(value=s) for s in xs]) for xs in suggestions}
```

Comment:
the addition of a new line seems to be the reason there's a diff between these two lines.